Workbooks/AzureFirewallWorkbook.json

```diff
@@ -187,14 +181,265 @@
             },
             "queryType": 1,
             "resourceType": "microsoft.resourcegraph/resources"
+          },
+          {
+            "id": "13d9f6e9-9290-4b1e-b0d0-a7c0ebdc1aed",
+            "version": "KqlParameterItem/1.0",
+            "name": "Help",
+            "label": "Show Help",
+            "type": 10,
+            "description": "This will show some help information to help you understand the page you are on",
+            "isRequired": true,
+            "typeSettings": {
+              "additionalResourceOptions": []
+            },
+            "jsonData": "[\r\n { \"value\": \"Yes\", \"label\": \"Yes\"},\r\n {\"value\": \"No\", \"label\": \"No\", \"selected\":true },\r\n { \"value\": \"Change Log\", \"label\": \"Change Log\"}\r\n]"
+          },
+          {
+            "id": "68f2cb32-03d4-4307-8eb5-1f66dfc09a85",
+            "version": "KqlParameterItem/1.0",
+            "name": "FirewallResources",
+            "label": "Show Resources",
+            "type": 2,
+            "description": "The is only used on the Overview page.",
+            "typeSettings": {
+              "additionalResourceOptions": [],
+              "showDefault": false
+            },
+            "jsonData": "[\r\n { \"value\": \"Yes\", \"label\": \"Yes\", \"selected\":true},\r\n {\"value\": \"No\", \"label\": \"No\" }\r\n]",
+            "value": "No"
+          },
+          {
+            "id": "53aa8daf-29de-40a1-9147-c0970e32fac4",
+            "version": "KqlParameterItem/1.0",
+            "name": "GeoLocation",
+            "label": "GeoLocationCSV",
+            "type": 2,
+            "description": "Supplied as a value for GeoLocation lookup on queries: Read more here - https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/ipv4-lookup-plugin#ipv4-lookup---matching-rows-only",
```

Comment:
Can you please remove the locale from the documentation link here and all its occurrences in this file? This is the reason for Validation failure.

Use this link : **https://docs.microsoft.com/azure/data-explorer/kusto/query/ipv4-lookup-plugin#ipv4-lookup---matching-rows-only**

Reply:
@v-rucdu,

Done.